src/reducer.js

```diff
@@ -9,7 +10,7 @@ const injectQuery = (location) => {
     return location
   }
 
-  const searchQuery = location.search || window.location.search
+  const searchQuery = location.search || (isBrowser && window.location.search)
```

Comment:
I think instead of add a new a package (`isBrowser`) I'd suggest the following:

```javascript
if (!location && typeof window === 'undefined') {
  return { ...location, query: {} };
}

const searchQuery = location.search || window.location.search
if (typeof searchQuery !== 'string' || searchQuery.length === 0) {
    return { ...location, query: {}  }
}
```

Reply:
I checked package before adding, this package is too small and solve our problem. Some ElectronJS applications include window as global variable, which contains BrowserWindow instance. So this code will throw error.